python/cugraph/dask/structure/renumber_wrapper.pyx

```diff
@@ -88,33 +91,80 @@ def mg_renumber(input_df,
 
     cdef bool is_hyper_partitioned = False # for now
 
+    cdef uintptr_t shuffled_major = <uintptr_t>NULL
+    cdef uintptr_t shuffled_minor = <uintptr_t>NULL
+    
+    cdef bool do_check = False # ? for now...
+    cdef bool mg_flag = is_multi_gpu # run Single-GPU or MNMG
+
+    cdef pair[unique_ptr[device_buffer], size_t] pair_original
+    cdef pair[unique_ptr[device_buffer], size_t] pair_partition
+
+    cdef unique_ptr[major_minor_weights_t[int, float]] ptr_shuffled_32_32
+    cdef unique_ptr[major_minor_weights_t[int, double]] ptr_shuffled_32_64
+
+    cdef unique_ptr[renum_quad_t[int, int]] ptr_renum_quad_32_32
+    cdef unique_ptr[renum_quad_t[int, long]] ptr_renum_quad_32_64
+    cdef unique_ptr[renum_quad_t[long, long]] ptr_renum_quad_64_64
+
+    cdef unique_ptr[vector[int]] uniq_partition_vector_32
+    cdef unique_ptr[vector[long]] uniq_partition_vector_64
+
+    cdef size_t rank_indx = <size_t>rank
+    
     if (vertex_t == np.dtype("int32")):
```

Comment:
Feels like we should raise an exception if any of these 3 ifs fail (at least for now) rather than silently failing to do anything leaving the impression that it worked.

Reply:
Yes. In fact I'm in the process of adding the missing instantiations in both `cython.cu` and `renumber_wrapper.pyx`. And I'll add an exception for the `else:` branch

Reply:
@seunghwak , the above triggered an issue: `renumber_edgelist()` is explicitly instantiated _only_ for `<int32_t, int32_t, true | false>`. As a result, I cannot instantiate renumbering C++2Cy interface (`call_renumber()` in `cython.cu`) for anything else, either.

Was that the intended functionality for `renumber_edgelist()`, or an omission?

Reply:
I added them in one of my other PRs.  I suspect this was an omission.

Reply:
Yes, they need to be added.

Reply:
Okay, for the sake of moving along, I'll just add the missing `renumber_edgelist()` instantiations. Going once, going twice,...

Reply:
Done: handled missing instantiation paths in both C++ and Cython. And added exception raising on un-handled paths in Cython.